dracut/30afterburn/afterburn-hostname.service

```diff
@@ -16,4 +16,6 @@ OnFailureJobMode=isolate
 
 [Service]
 ExecStart=/usr/bin/afterburn --cmdline --hostname=/sysroot/etc/hostname
+ExecStart=/bin/sh -c 'mkdir -p /sysroot/run/tmpfiles.d'
```

Comment:
This should be just `/run/tmpfiles.d`. The `/run` tmpfs is carried over from the initrd at switchroot.